src/extension.ts

```diff
@@ -24,6 +69,28 @@ export async function activate(context: vscode.ExtensionContext) {
                                      context.workspaceState);
       }));
 
+  // Create a command to open the project root .clangd configuration file.
+  context.subscriptions.push(
+      vscode.commands.registerCommand('clangd.projectSettings', () => {
+        if (vscode.workspace.workspaceFolders) {
```

Comment:
can we be a bit more explicit about these conditions? I think we're really testing length == 0 here, right?